rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/users/User.java

```diff
@@ -20,8 +16,11 @@
 import javax.persistence.TemporalType;
 import javax.validation.constraints.NotNull;
 import javax.validation.constraints.Size;
-import java.util.Date;
-import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'lombok\.NoArgsConstructor' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.